src/controllers/users.controller.js

```diff
@@ -0,0 +1,79 @@
+const usersServices = require('../services/users/users.services');
+
+const getUsers = (_, res) => {
+  return res.status(200).json(usersServices.getUsers());
+};
+
+const getUserById = (req, res) => {
+  const { id } = req.params;
+
+  const userId = Number(id);
+
+  if (!userId || isNaN(userId)) {
+    return res.sendStatus(400);
+  }
+
+  const user = usersServices.getUserById(userId);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+  res.send(user);
+
+  return !user ? res.sendStatus(404) : res.status(200).json(user);
```

Comment:
The line `res.send(user);` is redundant because the next line already handles the response. Consider removing it to avoid sending multiple responses.